### PR TITLE
1.13 Updates/Additions

### PR DESCRIPTION
**Pull request by katiemacaulay**
This PR covers some of the changes and updates from 1.12 to 1.13. 

- Instructions for UE version 4.25.6
- Instructions include a link to prerequisites- can be easy to skip
- Some updates to Viewport camera to help guide users who want multiple viewports
- Remove lane spline information but updated the import/export road network widget
- Updated scenario editor/ scenario creation with new pictures and updated info
- Created Vehicle Behavior Trigger Box Page
- Updated Scenario Widget Tool as well. 
- Made some updates to front page text.

Questions/ Suggestions Needed

- Do we want to reorganize the scenario file creation process into a different side panel structure/.md file structure?
- The front page paragraph text needs some updating, please look over/ suggestions welcomed.
- Update camera models configurations that can have a viewport?

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Installation
+
+ - **Build Errors**
+     - Ensure that you have followed all the instructions in [getting started](Getting_Started.md). 
+     - Cuda is a requirement to run the simulator, see [prerequisites](Getting_Started.md).
```
Comment (plarios) [inline docs/FAQ.md:5]: the CUDA SDK is a requirement for the scenario editor, not for the simulator

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+ - **Long Loading Time**
+    - monoDrive's Simulator has high quality graphics that will take some time to compile
```
Comment (plarios) [inline docs/FAQ.md:8]: this also applies only to the scenario editor

File: docs/scenario_editor/scenarios.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ controlling various different aspects of AI vehicle behavior. There are two type
```
Comment (citlaligm) [inline docs/scenario_editor/scenarios.md:5]: Need to change the above paragraph:
"The monoDrive Scenario Editor allows users to create their own scenarios by controlling various different aspects of AI vehicle behavior. There are two types of files that can be created: the trajectory file for Replay Mode and the scenario file for Closed loop mode."

Try something like this (feel free to change it):
"The monoDrive Scenario Editor allows users to create their own scenarios by controlling various different aspects of AI vehicle behavior, these scenarios are used when running the simulator or editor on closed loop mode"

File: docs/scenario_editor/trigger_boxes.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<div class="img_container">
+    <img class='md_img' src="../imgs/trigger_scene1.png"/>
+    <div class="space"></div>
+    <img class='md_img' src="../imgs/trigger_scene2.png"/>
+</div>
+
+
+## Adding a Trigger Box
+
+1. In the vehicle's "Details" window, click the "Add Component" button at the top and search for "Vehicle Behavior Trigger."
```
Comment (citlaligm) [inline docs/scenario_editor/trigger_boxes.md:17]: I think we should add something like:
"Click the vehicle and then go the "Details" window"

If this the first time user does this, can be a bit strange not to know where is the details window.

File: docs/scenario_editor/trigger_boxes.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  - **Distance:** The distance forward or backward (+/-) from the vehicle to place the trigger box. 
+
+  - **Lane:** The number of lanes to shift the trigger box. 
+  Negative numbers indicate lanes to the left of the vehicle, and positive to the right. 
+
+  - **Scale:** How large the trigger box is.
+
+  - **Owner:** The description of the actions that this box will trigger on the owning vehicle.
+
+  - **Other:** "The description of the actions that this box will trigger on the other vehicle.
```
Comment (citlaligm) [inline docs/scenario_editor/trigger_boxes.md:48]: Remove extra quotes

File: docs/scenario_editor/trigger_boxes.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+  - **Change Lane Right:** Triggers the vehicle to execute a lane change to the right lane if one is available. 
+
+  - **Can Stay in Lane:** If selected and multiple lane change options are selected the vehicle may chose to stay in its lane. 
+
+  - **Next Turn Left:** Triggers the vehicle to take the leftmost lane at the next intersection or lane branch. 
+
+  - **Next Turn Right:** Triggers the vehicle to take the rightmost lane at the next intersection or lane branch.
+
+  - **Next Turn Center:** Triggers the vehicle to take the center most lane at the next intersection or lane branch.
```
Comment (citlaligm) [inline docs/scenario_editor/trigger_boxes.md:61]: Is this correct "center most lane"? may a typo from copy-paste? If so, probably an error on the simulator description, too?

File: docs/scenario_editor/scenario_files.md [hunk 7]
```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
 
 The AI vehicle behavior details and tags that can be altered to create the scenario file can be found in [Scenario Creation](../scenarios)
```
Comment (citlaligm) [inline docs/scenario_editor/scenario_files.md:5]: I feel like this file and the **scenarios** should be merged one section called Scenario File Creation (similar to the Trajectory File Creation)  and there should be a sub-section for the  Scenario Editor and a subsection for the Scenario Setup. What do you think? Or at least have these sections one after another on the left bar.
image

File: docs/scenario_editor/scenario_files.md [hunk 8]
```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
...
 
-## Scenario Tool Widget
+## Scenario File Tool Widget
 
 To begin, find the under the Content Folder.
```
Comment (citlaligm) [inline docs/scenario_editor/scenario_files.md:9]: "find the tool under the Content Folder, right-click and select Run Editor Utility Widget "

File: docs/scenario_editor/scenario_files.md [hunk 9]
```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
...
 
   <div class="img_container">
     <img class='lg_img' src="../imgs/scenario_tool_widget.png"/>
   </div>
 
-Selecting "Save" underneath the Output path will write the scene's current configuration file to the path indicated in the text field. This will overwrite any paths that have the same name. Input path will load the configuration from the path given in the input path text field.
+In the "Scenario File Tool Widget," the user can load or save a new scenario file. 
+Selecting "Save" will write the scene's current configuration file indicated after FilePath. This
```
Comment (citlaligm) [inline docs/scenario_editor/scenario_files.md:16]: indicated on the specified file on FilePath, may be?

File: docs/scenario_editor/scenario_files.md [hunk 10]
```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
...
 To begin, find the under the Content Folder.
 
   <div class="img_container">
     <img class='lg_img' src="../imgs/scenario_tool_widget.png"/>
   </div>
 
-Selecting "Save" underneath the Output path will write the scene's current configuration file to the path indicated in the text field. This will overwrite any paths that have the same name. Input path will load the configuration from the path given in the input path text field.
+In the "Scenario File Tool Widget," the user can load or save a new scenario file. 
+Selecting "Save" will write the scene's current configuration file indicated after FilePath. This 
+will overwrite any paths that have the same name. Clear all will remove all loaded
```
Comment (citlaligm) [inline docs/scenario_editor/scenario_files.md:17]: "will overwrite any file that has the same name."?

File: docs/scenario_editor/scenario_files.md [hunk 11]
```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
...
+In the "Scenario File Tool Widget," the user can load or save a new scenario file. 
+Selecting "Save" will write the scene's current configuration file indicated after FilePath. This 
+will overwrite any paths that have the same name. Clear all will remove all loaded 
+scenario assets.
+
 
   <div class="img_container">
-    <img class='lg_img' src="../imgs/export_tool.png"/>
+    <img class='lg_img' src="../imgs/scenario_tool_widget2.png"/>
   </div>
```
Comment (citlaligm) [inline docs/scenario_editor/scenario_files.md:23]: Can we add a note that, the user needs to Clear All assets on the editor before using any client to playback the scenario? Failing to do that, will cause the vehicles to fly all over the place.

**Review by citlaligm — CHANGES_REQUESTED** [review]
Katie, great job getting all these changes and updates! Just some small changes and adjustments. Looking great!

**Comment by katiemacaulay** [comment]
@citlaligm, ready for another look. thank you for your help!!

**Review by citlaligm — APPROVED** [review]
Looks ready to go!

**Pull request merged**